ang/civicase.ang.php

```diff
@@ -235,6 +235,14 @@
 $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
 $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
 
+// Retrieve civicase webform URL
+$allowCaseWebform = Civi::settings()->get('civicaseAllowCaseWebform');
+$webformURl = NULL;
+if ($allowCaseWebform) {
+  $webformURl = Civi::settings()->get('civicaseWebformUrl');
+}
+$options['webformUrl'] = $webformURl;
```

Comment:
I think `newCaseWebformUrl` would be a bit more explicit, unless this is used for something else other than creating new cases.